pylicense3/cli.py

```diff
@@ -183,17 +183,20 @@ def write_header(
     # license_str
     l_str = '\n{}'.format(prefix).join(license_str.split('\n'))
     target.write(u'{} License: {}\n'.format(prefix, l_str))
-    # authors
-    target.write(prefix + ' Authors:\n')
-    max_author_length = 0
-    for author in authors:
-        max_author_length = max(max_author_length, len(author))
-    for author in sorted(authors.keys()):
-        year = '(' + authors[author] + ')'
-        if len(prefix) + 4 + max_author_length + len(year) <= max_width:
-            for ii in range(max_author_length - len(author)):
-                author += ' '
-        target.write(u'{}   {} {}\n'.format(prefix, author, year))
+    # authors, either as dict with years or only a contribution team
+    if isinstance(authors, str):
+        target.write(prefix + f' Contributors team: {authors}\n')
```

Comment:
What do you think about giving the user even more flexibility here? 
```suggestion
        target.write(f' {prefix} {authors}\n')
```
So not even the "Contributors team" would be hard coded.